akka-http/src/main/scala/akka/http/javadsl/server/Directives.scala

```diff
@@ -26,4 +28,96 @@ object Directives extends AllDirectives {
 
   @varargs override def getFromBrowseableDirectories(directories: String*): Route =
     super.getFromBrowseableDirectories(directories: _*)
+
+  /**
+   * Partially applying functions for the Java World!
```

Comment:
Actually, isn't the act of partially applying called [_currying_](https://en.wikipedia.org/wiki/Currying) or am I mixing sth up? In that case, `curry` would be a more apt method name.